src/api/nodeRpcCall.js

```diff
@@ -2489,8 +2489,13 @@ const senateProposeCancel = async (walletAddress, idx, executionBlock) => {
   const api = await getApi();
   const threshold = await senateMajorityThreshold();
   const executeProposal = api.tx.scheduler.cancel(executionBlock, idx);
-  const extrinsics = await createSenateProposalAndVote(threshold, executeProposal, true);
-  const extrinsic = api.tx.utility.batchAll(extrinsics);
+  const [proposal, voteAye] = await createSenateProposalAndVote(threshold, executeProposal, true);
+
```

Comment:
This is where that function should be used, yes ?

Reply:
yes, if there is only one senate member, it execute proposal without voting